tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1705,9 +1766,11 @@ tensorflow::Status ConvertConv2DHelper(OpConverterParams* params,
                                        ConvolutionType type) {
   switch (type) {
     case ConvolutionType::DEFAULT:
-      return ConvertConv2DHelper(params, 1);
+      return ConvertConv2DHelper(params, 1, /*is_deconv=*/false);
     case ConvolutionType::DEPTHWISE_CONV:
-      return ConvertConv2DHelper(params, 0);
+      return ConvertConv2DHelper(params, 0, /*is_deconv=*/false);
+    case ConvolutionType::TRANSPOSE:
+      return ConvertConv2DHelper(params, 1, /*is_deconv=*/true);
```

Comment:
should we name it is_transpose instead of is_deconv everywhere to have consistent terminology?

Reply:
Yes I would like to have to it be consistent. However we also have the need_transpose bool in this function used for transposing between NHWC and NCHW. I thought it might be confusion to have is_transpose as well. Perhaps `is_conv2dbackpropinput` or `is_conv2d_transpose`?

Reply:
I slightly prefer is_conv2d_backprop_input just because that is derived from the op name and we are not using a different terminology. conv2d_transpose seems good as well, if we expect all readers of this code familiar with that. conv2d_transpose is also shorter. So, feel free to choose either of it.